lms/djangoapps/teams/static/teams/js/views/my_teams.js

```diff
@@ -5,9 +5,19 @@
         function (Backbone, gettext, TeamsView) {
             var MyTeamsView = TeamsView.extend({
                 render: function() {
-                    TeamsView.prototype.render.call(this);
-                    if (this.collection.length === 0) {
-                        this.$el.append('<p>' + gettext('You are not currently a member of any teams.') + '</p>');
+                    var self = this;
+                    if (this.collection.isStale) {
+                        this.$el.html('');
+                        this.collection.fetch()
+                            .done(function() {
+                                self.collection.isStale = false;
+                                self.render();
+                            });
+                    } else {
+                        TeamsView.prototype.render.call(this);
```

Comment:
Why are we only calling the prototype's render if the collection isn't stale? Does this get called eventually in the stale case?